mautic/mautic
# feat: [DPMMA-2732] Send notification change password

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

The functionality adds when editing user data such as e-mail address and password, notifications via e-mail with information about changing the password or e-mail address for the user to whom the account applies.

![image](https://github.com/mautic/mautic/assets/39382654/e6506af0-cb43-4265-8732-0e6362268094)


Additionally, as part of the task, a bug related to the lack of redirection when trying to edit a user with an incorrect ID was corrected
![1](https://github.com/mautic/mautic/assets/39382654/861d7579-3a79-4248-816f-44f3d92c0fdc)

![2](https://github.com/mautic/mautic/assets/39382654/fb2ea30e-acfc-400a-af54-c68e016198e9)






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to the https://mautic5.ddev.site/s/users 
3. Go to user edit https://mautic5.ddev.site/s/users/edit/8
4. Change password or email
5. A notification email should be sent to the user
